Components/DB/Dbal/DbalStepsProvider.php

```diff
@@ -291,4 +291,12 @@ protected function getConnection($context)
 
         return $this->doctrine->getConnection($connectionName);
     }
+
+    /**
+     * @param $context
+     */
+    protected function rollback(&$context)
+    {
+        $this->getConnection($context)->executeQuery('ROLLBACK', [], []);
```

Comment:
Doctrine DBAL has a method for the rollback. https://www.doctrine-project.org/projects/doctrine-dbal/en/2.8/reference/transactions.html

Reply:
yes. I will remove from here. We will handle elsewhere.